pipe_naf_reader/naf_parser.py

```diff
@@ -91,7 +91,7 @@ def process(self, name, input_stream, output_stream):
                 logging.error(err)
                 logging.error("Unable to convert NAF message to csv row at {}".format(stripped_line))
                 #if it is not valid just exclude it.
-                pass
+                exit(1)
             except:
                 pass
```

Comment:
@smpiano what's this about? I think we should add some logging here and and `exit(1)` also